posts/innovative-snowflake-features.md

```diff
@@ -0,0 +1,208 @@
+---
+authors:
+- Pooja Krishnan
+tags:
+- Cloud
+- Snowflake
+date: 2019-08-07T10:33:00.000Z
+title: "Innovative Snowflake Features"
+image:
+---
+I recently attended the Snowflake Partner Bootcamp in Baltimore, and was given the opportunity to learn more about Snowflake. In the following blog, I am going to give a run-down of the innovative features Snowflake offers to its clients and some of the benefits of using them.
+
+# What is Snowflake?
+[Snowflake](https://www.snowflake.com/) is a Data Warehouse as a Service in the cloud. Highly available and fully managed, Snowflake handles the following for you:
+* Authentication
+* Configuration
+* Resource Management
+* Data Protection
+* Availability
+* Optimization
+* Cross Cloud Support
+
+Snowflake runs on popular cloud providers, including Microsoft Azure, Amazon Web Services and, as of July 2019, Google Cloud Platform (available in Public Preview).
+
+Snowflake also provides native connectors for a variety of languages in addition to a Web UI and Command Line Interface called SnowSQL.
+
+## Snowflake Editions
+Snowflake offers a variety of editions which offer differing levels of service. Look at [Snowflake Editions](https://docs.snowflake.net/manuals/user-guide/intro-editions.html "Snowflake Editions") in the Snowflake Documentation for more information.
+
+---
+# Why Snowflake?
+Snowflake uses a central data repository for persisted data that is accessible to all compute nodes and processes queries in MPP (Massively Parallel Processing) compute clusters referred to as Virtual Warehouses (we'll address these more later). This unique architecture allows both for easy data management as well as performance benefits and increased scalability.
```

Comment:
Usually you list out all the words in an acronym first, then provide the acronym.  Like: "Massively Parallel Processing (MPP)".  Typically you do this if you use the acronym in other places in the article.  Otherwise I'd leave it out.